indra/tests/test_pybel_api.py

```diff
@@ -409,6 +409,27 @@ def test_phosphorylation_one_site_with_evidence():
     assert ev.epistemics == {'direct': True, 'section_type': 'abstract'}
 
 
+def test_doi_evidence():
```

Comment:
This test errors on Github actions, is this due to a PyBEL version issue (like this code assumes a newer version than what is installed there)?
```
  File "/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/site-packages/pybel/struct/graph.py", line 918, in <dictcomp>
    for key, values in annotations_dict.items()
  File "/opt/hostedtoolcache/Python/3.6.13/x64/lib/python3.6/site-packages/pybel/struct/graph.py", line 954, in _clean_value
    raise NotImplementedError(f'where is key {key}?')
NotImplementedError: where is key TextLocation?
```

Reply:
Fixed - this happened because there was missing metadata when I created the BELGraph class